### DISCUSSION TITLE
SentryFlutter Performance Monitoring not working

### DISCUSSION THREAD

**Discussion by thertzelle** [getsentry/sentry-dart#719] [Q&A]
Here is my configuration in main: (I removed my DSN)

```
await SentryFlutter.init(
      (options) => {
        options.debug = true,
        options.dsn =
            '',
        options.tracesSampleRate = 1.0,
        options.tracesSampler = (samplingContext) {
          return 1;
        }
      },
      appRunner: () => runApp(App()),
    );
```

But when I run my app, I get the following message:
```
Sentry - debug:: No tracesSampleRate and tracesSampler set. Will not start SentryPerformanceTrackingIntegration.
```

I tried to look everywhere to see if I needed something else enabled, but the documentation doesnt say anything additional...

**Comment by marandaneto** [comment]
Answered on https://github.com/getsentry/sentry-dart/issues/720
Please open only the discussion or the issue, not both.
